### ISSUE TITLE
Question: how to force recipe upload

### ISSUE THREAD

**Issue by pananton** [conan-io/conan-package-tools#547]
### Question: how to force recipe upload

Hello! We use CPT in our CI scripts. During CI we build packages for several types of OS (Win + Linux) and when the CPT tries to upload packages it sometime fails to do this with error "Upload recipe to 'my_remote' failed: Remote recipe is newer than local recipe" . Looks like the problem is in the fact that recipes for distinct platforms are built and uploaded concurrently from distinct machines. I know that there is a parameter `--force` for `conan upload` command but I can't find a way to set it in current CPT version. Can you give some advice how this can be fixed? Maybe the intended design is to serialize build process to handle distinct OSes one after another?

**Comment by uilianries** [comment]
> Remote recipe is newer than local recipe" . Looks like the problem is in the fact that recipes for distinct platforms are built and uploaded concurrently from distinct machines. 

True. This is a known case.

>  know that there is a parameter --force for conan upload command but I can't find a way to set it in current CPT version. 

CPT uses Conan API to upload all packages. The API changes some parameters, including `force`, not it's a build policy. I'll open a PR fixing it. Thanks for reporting.

**Comment by pananton** [comment]
Thanks! Can you also tell me when this fix will be released approximately? Will it be included in next release (0.36) or maybe current (0.35) will be updated somehow (I'm not from Python world and have no idea, how pip packages are updated).

**Comment by uilianries** [comment]
This current week, or next week. CPT doesn't follow a specific schedule like Conan for releases. If we found a bug or a good feature, we release ASAP.

**Comment by pananton** [comment]
Can you please also explain why I can see following lines in CPT output when building package:
```
Uploading package 1/4: baeabe89039dbfe15c356d93e1df344efe21b3a6 to 'my_remote'            
Package is up to date, upload skipped
```

I remove all packages and recipe from *my_remote* before performing CPT build, still I can see this message at the end of CPT output. Note, that packages actually **are** uploaded, but CPT output is confusing.

**Comment by uilianries** [comment]
Fixed by #548

It should be available on CPT 0.35.1

**Comment by pananton** [comment]
Can you please tell when 0.35.1 will be available for download?
